Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [workspace]
 members = [
   "src/bootstrap",
-  "src/rustc",
-  "src/librustc_codegen_llvm",
+  "compiler/bin",
+  "compiler/rustc_codegen_llvm",
```

Comment:
`rustc_codegen_llvm` is no longer necessary here, AFAIK.

Reply:
Indeed, you are right.